Sources/App/Utilities/Environment+dotenv.swift

```diff
@@ -0,0 +1,77 @@
+//
+// Copy of extension from vapor-community/vapor-ext
```

Comment:
If we're going with an adapted version maybe this should read "Adapted from vapor-community/vapor-ext" or similar

Reply:
updated this